### PR TITLE
Issue/46

### PR DESCRIPTION
**Pull request by aloistr**
I am done with writing pods and making tests.
Some functions may be untested, but not important ones.
Please review and merge (this branch is already merged with branch documentation)

### CODE REVIEW

**Comment by kevindecapite** [comment]
I will review all of this during my afternoon. I am currently performing code reviews on another project all morning. Thank you.

**Comment by aloistr** [comment]
Good. It is quite impractical to have 9 hour time difference.  My workday is over (7 pm) when yours just began.

**Comment by kevindecapite** [comment]
I know. That is one reason I started staying up later to have some overlap of hours with your early morning.

**Review by kevindecapite — COMMENTED** [review]
~~For our reference, the following functions do not have tests:~~
I added the following tests, for completeness sake:

- [x] `swe_gauquelin_sector()`
- [x] `swe_heliacal_ut()`
- [x] `swe_heliacal_pheno_ut()`
- [x] `swe_vis_limit_mag()`
- [x] `swe_nod_aps()`
- [x] `swe_nod_aps_ut()`

**Comment by kevindecapite** [comment]
I applied my additions so feel free to merge and delete this after you review the latest commits.

**Review by aloistr — COMMENTED** [review]
OK

File: swephp.c [hunk 1]
```diff
@@ -3172,18 +3493,58 @@ PHP_FUNCTION(swe_refrac)
...
+  [0..3]	 array of 4 doubles: 
+	[0] true altitude, if possible; otherwise input value
+	[1] apparent altitude, if possible; otherwise input value
+	[2] refraction
+	[3] dip of the horizon
+
+
+=head3 C declaration
+
+  double swe_refrac_extended(double inalt, double geoalt, double atpress, double attemp, double lapse_rate, int32 calc_flag, double *dret)
```
[the remark this answers is not in the document]
Comment (aloistr) [reply swephp.c:3553]: The C library has double CALL_CONV swe_refrac_extended(double inalt, double geoalt, double atpress, double attemp, double lapse_rate, int32 calc_flag, double *dret)
which is correct.

**Review by aloistr — COMMENTED** [review]
ok

File: swephp.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (kevindecapite) [reply swephp.c:3553]: Yes, I saw that as well. Just an error in the docs then:

Screen Shot 2021-07-23 at 10 18 59 PM

**Pull request merged**